docs/developer/releasing.md

```diff
@@ -11,6 +14,23 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Prerequisites
+
+### `gcloud`
+
+With gcloud, run `gcloud auth login` and select your account [listed here](https://github.com/kubernetes/k8s.io/blob/05ada8c9ff90e7921e10d86ac5d59f5c1f4f74dc/groups/groups.yaml#L113). Open a PR if your account is not listed there but you believe it should be.
+
+### `docker`
+
+Enable the experimental features for the docker CLI by adding
+
+```json
+"experimental" : "enabled"
```

Comment:
I normally just set the env var `DOCKER_CLI_EXPERIMENTAL=enabled` rather than messing with the config. We used to set this in the Makefile, I'm wondering if we need to do that again instead?

Reply:
That's only valid on linux, while on mac, you'll need to enabled it through the daemon config

Reply:
We should do both though?

Reply:
Really? there doesn't seem to be a disclaimer about that in the docs: https://docs.docker.com/engine/reference/commandline/cli/#environment-variables

Reply:
oh huh, selecting the box in preferences restarts the daemon, but if you disable it and use this env var it appears to work...at least `docker manifest inspect` worked just fine in this situation.

Reply:
Maybe it was fixed 🤷‍♂

Reply:
yeah, i think it was, cause i read docs that a restart was required but i guess not any more!